Bottom: b19ca6ec945f9b6acea05d405a576158a542f2dc
Top:    242246a0af514b2dfb4e4c8c2caf20d32a432d75
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-25 14:08:14 +0100

Refresh of play-with-perspective

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 3c21498..7f09b69 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2063,8 +2063,8 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glMultMatrixf (scale);
 
   /* Translate to the center of the board space view */
-  glTranslatef (-(port->view_x0 + port->view_width / 2),
-                -(port->view_y0 + port->view_height / 2),
+  glTranslatef (-SIDE_X (port->view_x0 + port->view_width / 2),
+                -SIDE_Y (port->view_y0 + port->view_height / 2),
                 0.);
 
   /* Stash the model view matrix so we can work out the screen coordinate -> board coordinate mapping */
